### DISCUSSION TITLE
Warnings during testing

### DISCUSSION THREAD

**Discussion by ManiBharathiSS** [smartcontractkit/full-blockchain-solidity-course-py#1720] [Q&A] [answered]
Whenever I run tests, I get these warnings. What should I do to solve this?
![test_warning](https://user-images.githubusercontent.com/110660107/192503331-636ee49f-d09c-4c36-ae65-081cc12e4d97.png)

**Comment by cromewar** [comment, marked as answer]
Hello @ManiBharathiSS 
You don't have to worry about this, this is because the Web3.py has updated the way it manages some things and brownie still uses the old way, so we just have to wait until brownie updates itself to a newer version with that fixes. However as you can see they are not errors but warnings so for the moment you have nothing to worry about.

**Reply by ManiBharathiSS** [reply]
Thanks @cromewar :)
